src/components/dialogs/contingency-list-selector.jsx

```diff
@@ -146,30 +141,25 @@ const ContingencyListSelector = (props) => {
     };
 
     const removeFromFavorite = (toRemove) => {
```

Comment:
this function receive sometimes an array of ids, and another time an array of objects, for concistency with addFavorites use the same pattern

Reply:
Should be Array of Objects as arguments for addFavorites and removeFromFavorites now